qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/InactiveRouteStylingActivity.kt

```diff
@@ -95,32 +88,30 @@ class InactiveRouteStylingActivity : AppCompatActivity() {
         MapboxRouteLineApi(options)
     }
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(binding.root)
-
-        initNavigation()
-        initStyle()
-        initListeners()
-    }
-
-    override fun onStop() {
-        super.onStop()
-        mapboxNavigation.unregisterRouteProgressObserver(replayProgressObserver)
-        mapboxNavigation.unregisterRouteProgressObserver(routeProgressObserver)
-        mapboxNavigation.unregisterLocationObserver(locationObserver)
-        mapboxNavigation.stopTripSession()
-    }
+    private val mapboxNavigation by requireMapboxNavigation(
```

Comment:
This never loads unless you reference it. The lazy load can be confusing.

Also, the `onCreate` callback from activity happens before the `Lifecycle` onCreate. That causes a crash if you try to reference this delegate without using Lifecycle callbacks.

Reply:
Fixed it but was finding another race condition along the way.

```
mapboxNavigation.setRoutes(listOf(route))
binding.startNavigation.visibility = View.GONE
startSimulation(mapboxNavigation.getRoutes()[0]) <=== crashes because routes are not set yet
```